script.js

```diff
@@ -1,60 +1,45 @@
 const input = document.querySelector('#fruit');
-const suggestions = document.querySelector('.suggestions ul');
+const suggestionsList = document.querySelector('.suggestions ul');
 
 const fruit = ['Apple', 'Apricot', 'Avocado 🥑', 'Banana', 'Bilberry', 'Blackberry', 'Blackcurrant', 'Blueberry', 'Boysenberry', 'Currant', 'Cherry', 'Coconut', 'Cranberry', 'Cucumber', 'Custard apple', 'Damson', 'Date', 'Dragonfruit', 'Durian', 'Elderberry', 'Feijoa', 'Fig', 'Gooseberry', 'Grape', 'Raisin', 'Grapefruit', 'Guava', 'Honeyberry', 'Huckleberry', 'Jabuticaba', 'Jackfruit', 'Jambul', 'Juniper berry', 'Kiwifruit', 'Kumquat', 'Lemon', 'Lime', 'Loquat', 'Longan', 'Lychee', 'Mango', 'Mangosteen', 'Marionberry', 'Melon', 'Cantaloupe', 'Honeydew', 'Watermelon', 'Miracle fruit', 'Mulberry', 'Nectarine', 'Nance', 'Olive', 'Orange', 'Clementine', 'Mandarine', 'Tangerine', 'Papaya', 'Passionfruit', 'Peach', 'Pear', 'Persimmon', 'Plantain', 'Plum', 'Pineapple', 'Pomegranate', 'Pomelo', 'Quince', 'Raspberry', 'Salmonberry', 'Rambutan', 'Redcurrant', 'Salak', 'Satsuma', 'Soursop', 'Star fruit', 'Strawberry', 'Tamarillo', 'Tamarind', 'Yuzu'];
 
+const search = (inputValue) => fruit.filter(f => f.toLowerCase() === inputValue);
 
-function search(str) {
-	let results = [];
-
-	// Make fruit list lowercased
-	let fruitLowerCase = fruit.map(fruit => fruit.toLowerCase());
-
-	// Have user input lowercased
-	let inputLowerCase = str.toLowerCase();
-
-	// Loop through fruit to return fruits that includes user input string
-
-	for(i = 0; i < fruitLowerCase.length; i++){
-		if (fruitLowerCase[i].includes(inputLowerCase)){
-			results.push(fruitLowerCase[i]).toString;
-		};
-
-	};
-
-	return results;
-
+function clearSuggestions() {
```

Comment:
when a statement is repeated a lot, it better get isolated to avoid repeating yourself